src/EditorFeatures/Test2/Rename/InlineRenameTests.vb

```diff
@@ -1838,5 +1838,33 @@ End Class
                 Assert.Equal(InlineRenameFileRenameInfo.NotAllowed, session.FileRenameInfo)
             End Using
         End Sub
+
+        <WpfFact>
+        <Trait(Traits.Feature, Traits.Features.Rename)>
+        Public Sub VerifyFileRenamesCorrectlyWhenCaseChanges()
```

Comment:
Note that this doesn't actually validate the fix needed in this change, but I believe is a worthwhile test so I kept it.